Doc/library/xml.etree.elementtree.rst

```diff
@@ -853,10 +854,7 @@ Functions
    inclusions.  Limited to reduce the risk of malicious content explosion. Pass a
    negative value to disable the limitation.
 
-   Returns the expanded resource.  If the parse mode is
-   ``"xml"``, this is an ElementTree instance.  If the parse mode is "text",
-   this is a Unicode string.  If the loader fails, it can return None or
-   raise an exception.
+   Returns the expanded resource in *elem*.
```

Comment:
I think that using "returns" here is not correct, because the function always returns None.

Perhaps adding "in-place in *elem*" in the initial sentence "This function expands XInclude directives" would be enough.

Please update also the comment for `include()` in sources.

Reply:
@serhiy-storchaka, updated documentation and source comment as suggested